### PR TITLE
Fixed more missing / incorrect URDF elements.

### PR DESCRIPTION
**Pull request by danepowell**
Some of these are just wrapping up elements that I missed in https://github.com/PR2/pr2_common/pull/219 . The incorrect focal_lengths are clearly just due to rounding errors in how Gazebo calculates focal lengths- not a big deal but they throw a bunch of warnings.

### CODE REVIEW

File: pr2_description/urdf/sensors/wge100_camera.gazebo.xacro [hunk 1]
```diff
@@ -23,7 +23,11 @@
           <cameraName>${camera_name}</cameraName>
           <imageTopicName>${image_topic_name}</imageTopicName>
           <cameraInfoTopicName>${camera_info_topic_name}</cameraInfoTopicName>
+          <depthImageTopicName>${camera_name}/depth/image_raw</depthImageTopicName>
```
Comment (ahendrix) [inline pr2_description/urdf/sensors/wge100_camera.gazebo.xacro]: This isn't a depth camera; does it really need the depth camera tags?

**Comment by davetcoleman** [comment]
I have tested this PR by launching it and it seems to work on my machine

File: pr2_description/urdf/sensors/wge100_camera.gazebo.xacro (hunk 1 above)
Comment (danepowell) [inline pr2_description/urdf/sensors/wge100_camera.gazebo.xacro]: I'm not familiar with the camera, but it's implemented using the libgazebo_ros_depth_camera.so plugin, which does require these elements. Maybe it should be implemented using libgazebo_ros_camera.so instead?

**Comment by ahendrix** [comment]
It still smells funny. Why are we using the depth camera plugin to simulate a simple color camera?

**Comment by danepowell** [comment]
It looks like it was originally a plain old camera, but got changed to a depth camera two years ago with this commit - "for testing":
https://github.com/PR2/pr2_common/commit/4c071dd1fe7e1e1c2751933a170ad0589bd9dfa8#pr2_description/urdf/sensors/wge100_camera.gazebo.xacro

I don't know what was being tested, but it probably should have been changed back afterwards.

**Comment by ahendrix** [comment]
Can you change the wge100 plugin back to a regular camera and update this pull request?

File: pr2_description/urdf/sensors/wge100_camera.gazebo.xacro (hunk 1 above)
Comment (davetcoleman) [inline pr2_description/urdf/sensors/wge100_camera.gazebo.xacro]: Yes, it should. @hsu recently [changed](https://github.com/ros-simulation/gazebo_ros_pkgs/issues/82) this for the prisilica camera, but there are probably some other places that this needs to be changed.

**Pull request merged**